web/src/app/users/UserContactMethodList.js

```diff
@@ -34,6 +37,8 @@ const testCM = gql`
   }
 `
 
+const DISABLED_CM_TOOLTIP = `Number disabled. See contact method's options to reactivate`
```

Comment:
We should use something more generic than `Number` as future contact methods (e.g. email and/or webhooks) won't always be phone numbers.